client/state/stats/emails/selectors.js

```diff
@@ -165,7 +172,11 @@ export function getEmailStat( state, siteId, postId, period, statType ) {
  */
 export function getEmailStatsNormalizedData( state, siteId, postId, period, statType, date, path ) {
 	return state.stats.emails.items
-		? get( state.stats.emails.items, [ siteId, postId, period, statType, date, path ], null )
+		? get(
+				state.stats.emails.items,
+				[ ...getDataPath( siteId, postId, period, statType ), path ],
```

Comment:
As an example of my previous message, here you are not passing the `null` parameter.